bitrise/dependencies.go

```diff
@@ -165,27 +165,27 @@ func checkIsBitriseToolInstalled(toolname, minVersion string, isInstall bool) er
 		}
 		return doInstall()
 	}
-	verStr, err := command.RunCommandAndReturnStdout(toolname, "-version")
+	versionOutput, err := command.RunCommandAndReturnCombinedStdoutAndStderr(toolname, "-version")
 	if err != nil {
-		log.Infof("")
-		return errors.New("Failed to get version")
```

Comment:
I had some issues in CI and noticed that the real error was swallowed here -> now we print the combined output to help debugging the error